README.md

```diff
@@ -230,11 +230,15 @@ You can configure your dashboard for Basic Authentication by adding usernames an
       "user":"user2",
       "pass":"pass"
     }
-  ]
+  ],
+  "options": {
```

Comment:
Exactly. The whole config file is already a dict of options, it doesn't need another dict of options within it.